docs/src/main/sphinx/connector/hive.rst

```diff
@@ -484,7 +484,15 @@ with Parquet files performed by the Hive connector.
         definition. The equivalent catalog session property is
         ``parquet_use_column_names``.
       - ``true``
-
+    * - ``parquet.writer.page-size``
+      - Maximum page size for the Parquet writer.
+      - 1 MB
+    * - ``parquet.writer.block-size``
+      - Maximum block size for the Parquet writer.
+      - 128 MB
+    * - ``parquet.writer.batch-size``
+      - Maximum number of records passed to the writer in each batch.
```

Comment:
What's a batch?

should i, as a user, know that?

Reply:
Modified `batch` to `chunk` which fits the terminology of Parquet

https://parquet.apache.org/docs/concepts/

I don't know of a good and simple explanation for "batch". @mosabua  any hints?

Reply:
cc @raunaqmorarka @alexjo2144

Reply:
I don't think this batch size is related to parquet chunk, that is the name given to a column's data within a row group.
What's the end-user impact of this config ?
It's not clear from this what changing the batch size would do.
cc: @alexjo2144